src/HealthChecks.Aws.SecretsManager/SecretsManagerHealthCheck.cs

```diff
@@ -0,0 +1,85 @@
+using Amazon.SecretsManager;
```

Comment:
Please use file scoped namespaces for all new files.

Reply:
Fixed